CHANGELOG.md

```diff
@@ -1,6 +1,41 @@
 Changes
 =======
 
+# 5.15.0 / 07-10-2017
+**Linux, Windows, Docker and Source Install**
+
+### Details
+https://github.com/DataDog/dd-agent/compare/5.14.1...5.15.0
+
+### Changes
+Please refer to the [Integrations Core Checks 5.15.0 tag](https://github.com/DataDog/integrations-core/releases/tag/5.15.0) for the list of changes on the core checks.
+Please refer to the [Trace Agent 5.15.0 tag](https://github.com/DataDog/datadog-trace-agent/releases/tag/5.15.0) for the list of changes on the Trace Agent.
+
+* [FEATURE] [kube] Allow to run agent on the node [#3401][]
```

Comment:
Are you using `pimpmychangelog`? It automatically handles the PR and mentions to contributors - I don't see the links at the bottom of the changelog.md file so I'm assuming not. It's a gem super easy to install... grab it here: https://github.com/pcreux/pimpmychangelog :)